packages/@aws-cdk/aws-cloudfront/README.md

```diff
@@ -266,9 +270,24 @@ new cloudfront.Distribution(this, 'myDist', {
 });
 ```
 
+> **Note:** The above requires that the stack is in the `us-east-1` region.
+> Lambda@Edge functions must be created in the `us-east-1` region, regardless of the region of the CloudFront distribution and stack.
+> To make it easier to request functions for Lambda@Edge, the `EdgeFunction` construct can be used.
+> The `EdgeFunction` construct will automatically request a function in `us-east-1`, regardless of the region of the current stack.
+> `EdgeFunction` has the same interface as `Function` and can be created and used interchangably.
+
+```ts
+// `cloudfront.experimental.EdgeFunction` can be used in place of `lambda.Function`.
+// It will automatically create the `Function` in `us-east-1`, regardless of the current stack's region.
+const myFunc = new cloudfront.experimental.EdgeFunction(this, 'MyFunction', {
+  runtime: lambda.Runtime.NODEJS_10_X,
+  handler: 'index.handler',
+  code: lambda.Code.fromAsset(path.join(__dirname, 'lambda-handler')),
```

Comment:
I'd move the `EdgeFunction` example as the first one given this is the happy path for most users.